test/SILGen/objc_async_from_swift.swift

```diff
@@ -5,32 +5,30 @@ import Foundation
 import ObjCConcurrency
 
 @objc protocol SlowServing {
-    func requestInt() async -> Int
-    func requestString() async -> String
-    func requestIntAndString() async -> (Int, String)
+    //func requestInt() async -> Int
```

Comment:
Why is this all commented out?

Reply:
Sorry, I was doing that while I was testing before I had the code that broke the logic out into an async closure, so that I could check the SIL generated for individual thunks after hitting the "async functions can't be called from sync ones" assertion.